content_sync/pipelines/definitions/concourse/theme-assets-pipeline.yml

```diff
@@ -94,3 +110,27 @@ jobs:
           - 'Fastly-Key: ((fastly_live.api_token))'((purge_header))
           - https://api.fastly.com/service/((fastly_live.service_id))/purge/ocw-hugo-themes
   # END NON-DEV
+  # START NON-DEV
+  on_failure:
+    try:
+      do:
+      - put: slack-webhook
+        timeout: 1m
+        params:
+          alert_type: failed
+          text: |
+            Failed to build theme assets.
+            
+            Append `((atc-search-params))` to the url below for more details.
```

Comment:
This is...maybe not worth it.

The slack alert includes a URL, cool!

<img width="732" alt="Screen Shot 2022-11-17 at 4 20 48 PM" src="https://user-images.githubusercontent.com/9010790/202562295-8058d1f5-8bf0-42d5-97d7-ff1cbf0968ec.png">

But our pipeline names include some URL search parameters at the end, e.g.,
```
http://concourse:8080/teams/main/pipelines/ocw-theme-assets/jobs/build-theme-assets/builds/4?vars.branch=%22main%22
```
So the slack notification's URL doesn't actually work, which is annoying.

There doesn't appear to be a way to tell `slack-alert` resource to add query params. https://www.google.com/search?q=concourse+slack+alert&oq=concourse+slack+&aqs=chrome.0.69i59l2j69i57j0i512l2j69i60l3.1603j0j7&sourceid=chrome&ie=UTF-8

Reply:
These are called ["instance vars"](https://concourse-ci.org/instanced-pipelines.html) and yea, it doesn't seem that `slack-alert` supports them. It doesn't seem like there's a whole lot of activity over there, but I think if we think this important enough we could put up a PR there to add support for them, or at least create an issue.